src/components/ScholarshipForm.jsx

```diff
@@ -114,6 +146,96 @@ function ScholarshipForm({ scholarship, submitFn, onSubmitError }) {
     </Grid>
   );
 
+  if (experiments.expShowRequirementsSection) {
+    stepperItems['Eligibility Requirements'] = (
+      <Grid container spacing={3}>
+        <Grid item xs={12}>
+          <Typography className={classes.stepperDescription}>
+            Include information that is required for applicants to have.
+          </Typography>
+        </Grid>
+
+        <Grid item xs={12}>
+          <FormControlLabel
+            control={
+              <Checkbox
+                checked={hasReqs}
+                onChange={(event) => setHasReqs(event.target.checked)}
+                color="primary"
+              />
+            }
+            label="NO ELIGIBILITY REQUIREMENTS"
+          />
+        </Grid>
+
+        <Grid item sm={6} xs={12}>
+          <FormikMultiSelect
+            label="Grade(s)"
+            id="grades"
+            labelStyle={classes.inputLabel}
+            formik={formik}
+            options={gradeOptions}
+            placeholder="No grade requirements"
+          />
+        </Grid>
+
+        <Grid item sm={6} xs={12}>
+          <FormikTextField
+            label="Minimum GPA"
+            id="gpa"
+            formik={formik}
+            labelStyle={classes.inputLabel}
+          />
+        </Grid>
+
+        <Grid item sm={6} xs={12}>
+          <FormikAutocomplete
+            label="School(s)"
+            id="schools"
+            labelStyle={classes.inputLabel}
+            options={[...SCHOOLS]}
```

Comment:
Why not simply

```suggestion
            options={SCHOOLS}
```

Reply:
it throws the following error: `Uncaught TypeError: options.filter is not a function`. Probably because the `SCHOOLS` is a set

Reply:
Got it.